tests/bdd/common/deployer.py

```diff
@@ -1,34 +1,82 @@
 import os
 import subprocess
 
+from dataclasses import dataclass
+
+
+@dataclass
+class StartOptions:
+    mayastors: int = 1
+    wait: str = "10s"
+    csi: bool = False
+    reconcile_period: str = ""
+    cache_period: str = ""
+    mayastor_env: str = ""
+    agents_env: str = ""
+    jaeger: bool = True
+    extra_args: [str] = ()
+
+    def args(self):
+        args = [
+            "--mayastors",
+            str(self.mayastors),
+            "--wait-timeout",
+            self.wait,
+        ]
+        if self.csi:
+            args.append("--csi")
+        if self.jaeger:
+            args.append("--jaeger")
+        if len(self.reconcile_period) > 0:
+            args.append(f"--reconcile-period={self.reconcile_period}")
+            args.append(f"--reconcile-idle-period={self.reconcile_period}")
```

Comment:
Should we provide the ability to set the idle period independently?

Reply:
Could be, I left it as one for now because from code inspection we set them to the same.